packages/manager/modules/telecom-universe-components/src/telecom/telephony/accessories/telephony-accessories.constant.js

```diff
@@ -151,56 +144,47 @@ export const TELEPHONY_LINE_PHONE_ACCESSORIES = {
       'https://www.ovhtelecom.fr/images/telephonie/2020/poly/accessory-poly-mda220-90-90.png',
   },
   'sennheiser.mbpro1': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-mbpro1-90-90.png',
   },
   'sennheiser.mbpro2': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-mbpro2-90-90.png',
   },
   'sennheiser.mb660': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-mb660-90-90.png',
   },
   'sennheiser.sdw5036': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-sdw5036-90-90.png',
   },
   'sennheiser.sdw5066': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-sdw5066-90-90.png',
   },
   'sennheiser.sc268': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-sc268-90-90.png',
   },
   'sennheiser.sc238': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-sc238-90-90.png',
   },
   'sennheiser.sc165usb': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
     img:
       'https://www.ovhtelecom.fr/images/telephonie/2020/sennheiser/accessory-sennheiser-sc165usb-90-90.png',
   },
   'sennheiser.sc165usbc': {
-    url:
-      'https://www.ovhtelecom.fr/telephonie/comparatif-accessoires-telephone.xml/',
+    url: 'https://www.ovhtelecom.fr/telephonie/comparatif-des-accessoires.xml',
```

Comment:
Looks like this URL could be stored as a more global constant and just be used (instead of setting it for each reference). It will be easier to maintain